build/debs/BUILD

```diff
@@ -12,44 +19,74 @@ load("//build:workspace.bzl", "CNI_VERSION", "CRI_TOOLS_VERSION")
 # images.
 release_filegroup(
     name = "debs",
-    srcs = [
-        ":cri-tools.deb",
-        ":kubeadm.deb",
-        ":kubectl.deb",
-        ":kubelet.deb",
-        ":kubernetes-cni.deb",
-    ],
+    conditioned_srcs = for_platforms(
+        default = [],
+        for_client = [":kubectl-{ARCH}.deb"],
+        for_node = [
+            ":cri-tools-{ARCH}.deb",
+            ":kubeadm-{ARCH}.deb",
+            ":kubelet-{ARCH}.deb",
+            ":kubernetes-cni-{ARCH}.deb",
+        ],
+        only_os = "linux",
+    ),
+)
+
+# Create aliases from the non-arch names to the arch-specific names for backwards compatibility
```

Comment:
this only kinda works. the targets are still there, but the outputs moved AFAICT. 
https://github.com/kubernetes-sigs/kind/issues/336